README.rst

```diff
@@ -1,3 +1,5 @@
+.. highlight:: python
```

Comment:
Can you remove this line?

When I run this PR through `twine check`, it kicks this up:

```
> twine check dist/everett-1.0.4.dev0.tar.gz 
Checking dist/everett-1.0.4.dev0.tar.gz: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 1: Error: Unknown directive type "highlight".

    .. highlight:: python
```

`..highlight::` is a reST directive, so I'm not sure why readme_renderer doesn't like it. But we can skip that and just use `.. code-block:: [language]` across the file and that'd be fine and get us better highlighting on GitHub.